packages/terra-text/src/terra-dev-site/doc/example/TextFontSizes.jsx

```diff
@@ -3,6 +3,15 @@ import Text from 'terra-text';
 
 const TextFontSizes = () => (
   <div>
+    <p>
+      The `fontSize` prop can be used set the `font-size` on any `Text` component. The `fontSize` prop accepts one of the following numbers:
+      <br />
+      <br />
+      `10`, `12`, `14`, `16`, `18`, `20`, `24`, `32`, `36`, `46`, `50`, `56`, `64`, `72`, `92`, `100`
+      <br />
+      <br />
+      These values represent the pixel values of our UX defined font-scale. The `Text` component renders these font-sizes with `rem` units.
```

Comment:
Lets clean this description up to:
```
The font-size values represent the pixel values of our UX defined font-scale. Then the Tex` component will render these font-sizes with rem units.
```
and then place this description in the example render, not in the code:
```
// packages/terra-text/src/terra-dev-site/doc/text/Text.1.doc.mdx 
<TextFontSizes
   title="Font Sizes"
   description="<text from above>"
/>
```

Reply:
Updated here https://github.com/cerner/terra-core/pull/2886/commits/88d7739a51d61d65cec97299064fe301fcee4ce5.